src/main/app/case/answers/getAnswerRows.ts

```diff
@@ -24,7 +24,8 @@ export const getAnswerRows = function (section: Sections): GovUkNunjucksSummary[
     .filter(step => step.showInSection === section)
     .flatMap(step => {
       const fieldKeys = Object.keys(step.form.fields);
-      const stepContent = { ...this.ctx, ...step.generateContent({ isDivorce, partner, formState })[language] };
+      console.log(language);
+      const stepContent = { ...this.ctx, ...step.generateContent({ isDivorce, partner, formState }) };
```

Comment:
this part was just to push and show what the content files now look like. ill fix this after if the way is approved or requires changes.